src/LinAlg/Test/MatrixTests.h

```diff
@@ -0,0 +1,227 @@
+//==============================================================================
+//!
+//! \file MatrixTests.h
+//!
+//! \date Apr 11 2016
+//!
+//! \author Eivind Fonn / SINTEF
```

Comment:
Wrong author (and date)? (also for the C-file)

Reply:
it's copied from TestMatrix.C and as this code originates there, I'd say it is correct.